core/src/main/java/com/klaytn/caver/utils/Utils.java

```diff
@@ -0,0 +1,87 @@
+package com.klaytn.caver.utils;
+
+import org.bouncycastle.math.ec.ECPoint;
+import org.web3j.crypto.Hash;
+import org.web3j.crypto.Sign;
+import org.web3j.utils.Numeric;
+
+import java.util.regex.Pattern;
+
+public class Utils {
+    public static final int LENGTH_ADDRESS_String = 40;
+    public static final int LENGTH_PRIVATE_KEY_STRING = 64;
+    private static final Pattern HEX_STRING = Pattern.compile("^[0-9A-Fa-f]+$");
+
+    public static boolean isPrivateKeyValid(String privateKey) {
```

Comment:
How about changing the name to `isValidPrivateKey`?

Reply:
ok, I will modify it.